### DISCUSSION TITLE
osquery vulnerability management

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9302] [Unsupported Versions] [answered]
Hi!

I run SO 2.3.190 and I thought I give the vulnerability management features of osquery and Fleet a chance. The Linux hosts of my distributed SO environment are full of (old) vulnerabilities (even though I run OS updates regularly) and for my test Windows clients I don't get any vulnerabilities and it does not find all of my siftware (e.g. MS Office products are missing (no, I did not set the disable​_win​_os​_vulnerabilities option in the Fleet config). Any idea what I'm doing wrong? How reliable are these features of osquery and Fleet? Any experiences out there? Thanks in advance!

Cheers, Ben

**Comment by defensivedepth** [comment, marked as answer]
We still disable this by default for that reason - see here: https://github.com/Security-Onion-Solutions/securityonion/discussions/7643#discussioncomment-2440468
